react-web/src/pages/certification/Certification.tsx

```diff
@@ -248,7 +249,11 @@ const Certification = () => {
 
   return (
     <>
-      <div
+      {!subscribedFeatures?.length || subscribedFeatures?.indexOf("l1-run") === -1 ? 
```

Comment:
this should be enough 
```suggestion
      {subscribedFeatures?.indexOf("l1-run") === -1 ? 
```

Reply:
Done